### ISSUE TITLE
No ticks_position declared bunches Labels

### ISSUE THREAD

**Issue by waholleyiv** [seiyria/bootstrap-slider#417]
Upon load my ticks bunch themselves up with `width: 0px`, however upon clicking anywhere on the slider, the labels move into their 'normal expected positions' with `width: 70px`.  I have `ticks_positions` undeclared because it was forcing the slider to max out on click.  

Manually setting `.slider-tick-label{width:70px; !important}` works, but causes problems since the label positions can change depending on the starting hour.  The label positions will change with the starting hour (as expected) when the quantity of `ticks` changes, but only after they have returned to their 'normal expected positions'.  

Not sure what's going on here.
https://jsfiddle.net/xnu6rmwq/2/

**Bunched**
![screen shot 2015-08-10 at 10 52 40 pm](https://cloud.githubusercontent.com/assets/8657791/9189172/acba62b2-3fb2-11e5-9f00-06cd47538934.png)
**Normal Expected**
![screen shot 2015-08-10 at 10 52 48 pm](https://cloud.githubusercontent.com/assets/8657791/9189173/af4b60bc-3fb2-11e5-81f1-f842a45db026.png)
**Quantity of ticks dynamically changed after un-bunch**
![screen shot 2015-08-10 at 11 00 46 pm](https://cloud.githubusercontent.com/assets/8657791/9189226/a9669a6c-3fb3-11e5-87ba-7ca783bdc188.png)

**Comment by seiyria** [comment]
Please provide a jsfiddle as well.

**Comment by waholleyiv** [comment]
https://jsfiddle.net/xnu6rmwq/

**Comment by seiyria** [comment]
According to the documentation, you're [supposed to provide ticks_positions](https://github.com/seiyria/bootstrap-slider#options). If there's a bug with that behavior I think it should be handled separately, since this *technically* isn't valid, I think.

**Comment by waholleyiv** [comment]
@seiyria - I've updated my fiddle with the ticks_positions, but the auto-max bug now occurs.  Should I re-format this issue to reflect the bug or close this and open a new one?

**Comment by seiyria** [comment]
I think it's wise to open a new one with this new issue, since that's the heart of the issue here.
